spoon-pom/pom.xml

```diff
@@ -20,8 +20,7 @@
     <inceptionYear>2007</inceptionYear>
 
     <properties>
-        <java.src.version>1.8</java.src.version>
-        <java.test.version>1.8</java.test.version>
+        <maven.compiler.release>11</maven.compiler.release>
```

Comment:
what happens if we put "8" here as compiler target bytecode?

Reply:
The `target` flag does not set the bootstrap classpath correctly so things might break. Additionally, the JDT class files are compiled for Java 11, so lowering Spoon's target compatibility will have no real effect for end users.

Reply:
Java 11 source code cannot target Java 8. Java 9, 10 and 11 introduced several changes to the class file format, so 11 is binary incompatible with 8.

`javac` will refuse to compile if you try to set source to `1.8` and target to `11`.

What's more, we depend on JDT whose binaries are compiled for Java 11, which a Java 8 JVM cannot execute. There are simply no tricks we can pull here that would allow us to run Spoon on a Java 8 JVM.